src/acceptance/api/api_suite_test.go

```diff
@@ -11,9 +11,8 @@ import (
 	"testing"
 	"time"
 
-	"acceptance/config"
-	. "acceptance/helpers"
-
+	"code.cloudfoundry.org/app-autoscaler/src/acceptance/config"
```

Comment:
err ... not sure this is correct. The mod file should specify this better I think.